docs/source/Installation.md

```diff
@@ -199,10 +206,22 @@ Also, you need to set the environment variable `AWS_SQS` to `True` to activate t
 After having properly configured the environment files as suggested previously, you can run the image.
 The project uses `docker-compose`. You have to move to the project main directory to properly run it.
 
-`docker-compose up`
+`docker-compose up -d`
 
 
 ## After deployment
+
 ### Users creation
 You may want to run `docker exec -ti intel_owl_uwsgi python3 manage.py createsuperuser` after first run to create a superuser.
-Then you can add other users directly from the Django Admin Interface after having logged with the superuser account.
\ No newline at end of file
+Then you can add other users directly from the Django Admin Interface after having logged with the superuser account.
+
+> For Django Groups & Permissions settings, refer [here](https://intelowl.readthedocs.io/en/stable/Advanced-Usage.html#django-groups-permissions).
+
+## Update to the most recent version
```

Comment:
could you please update this section + the release notes with the steps to perform to update from previous versions? Can you also add the script required to be executed in the repo too?

Reply:
Done!